### ISSUE TITLE
crash with java.lang.NullPointerException

### ISSUE THREAD

**Issue by henry-lp** [SpoonLabs/sorald#124]
Spoon failed which in turn failed Sorald. 

Reproduction procedure after packaging the Jar file of Sorald:

```
git clone https://github.com/eclipse/rdf4j Repo4Analyze

java -jar target/sorald-1.1-SNAPSHOT-jar-with-dependencies.jar --originalFilesPath Repo4Analyze/ --ruleKeys 2164 --maxFixesPerRule 1
```
@fermadeiral @monperrus

**Comment by monperrus** [comment]
Still happens on the latest version?

**Comment by henry-lp** [comment]
@monperrus yes 
Here is the output

[Output.LOG](https://github.com/SpoonLabs/sorald/files/5577660/Output.LOG)

**Comment by slarse** [comment]
I'll look at this today

**Comment by monperrus** [comment]
great!

**Comment by slarse** [comment]
This is caused by duplication of classes in the `compliance` submodule of RDF4J, in combination with setting `env.setIgnoreDuplicateDeclaration(true)` in Spoon. For example, this test class is duplicated:

```
./compliance/rio/src/test/java/org/eclipse/rdf4j/rio/n3/N3ParserTest.java
./core/rio/n3/src/test/java/org/eclipse/rdf4j/rio/n3/N3ParserTest.java
```

In other words, this is a bug in Spoon, not sorald.

**Comment by slarse** [comment]
This should be fixed with INRIA/spoon#3709

**Comment by slarse** [comment]
This was indeed fixed by said PR.
